app/models/file.ts

```diff
@@ -172,11 +179,16 @@ export default class FileModel extends BaseFileItem {
                 resource: node.id,
                 ...options,
             }),
-            success: (data, _, xhr) => {
-                data.status = xhr.status;
+            success: (_, __, xhr) => {
+                status = xhr.status;
             },
         }).then(() => this.reload());
-        return req;
+        let returnObject = requestData;
+        if (typeof requestData !== 'object') {
```

Comment:
I don't understand why this is necessary. If the response is in json format, then `requestData` should always be an object. Is this done just to appease type checking, or do we actually encounter cases where `requestData` is a primitive instead of an object?

Reply:
Yea, for 202 requestData is an empty string